app/uk/gov/hmrc/customsservicestatus/controllers/CustomsServiceStatusController.scala

```diff
@@ -31,17 +31,19 @@ import scala.concurrent.ExecutionContext
 class CustomsServiceStatusController @Inject()(checkService: CustomsServiceStatusService, cc: ControllerComponents)(implicit ec: ExecutionContext)
```

Comment:
this val needs renaming from `checkService`

Reply:
renamed.